src/shared/lib/fetch/index.js

```diff
@@ -270,8 +270,44 @@ export const getArcGISCSVURLFromOrgId = async function(serverNumber, orgId, laye
  * @param {*} dashboardId the ID of the dashboard, as passed to the iframe that renders it (i.e. https://maps.arcgis.com/apps/opsdashboard/index.html#/ec4bffd48f7e495182226eee7962b422 is dashboardId = ec4bffd48f7e495182226eee7962b422)
  * @param {*} layerName the name of the layer to fetch data for, find this by examining requests
  */
-export const getArcGISCSVURL = async function(serverNumber, dashboardId, layerName) {
+export const getArcGISCSVURL = async (serverNumber, dashboardId, layerName) => {
   const dashboardManifest = await json(`https://maps.arcgis.com/sharing/rest/content/items/${dashboardId}?f=json`);
   const { orgId } = dashboardManifest;
   return getArcGISCSVURLFromOrgId(serverNumber, orgId, layerName);
 };
+
+/**
+ * Retrieves data from an ArcGIS REST API. By default, it will retrieve all items at the provided linked with geometry turned off.
+ * You can control pagination size through the `k` parameter in `options`
+ * @param {string} url URL of the resource
+ * @param {*} date the date associated with this resource, or false if a timeseries data
```

Comment:
Does this work with various and sundry data sources that store dates in different formats?

Reply:
All scrapers currently accessing ArcGIS json data will see a bunch of cache miss as the URL will change. So we will have to take into account. Regarding the source date format, this should not be a problem. The date field is for caching only (like the other fetch functions). The format of dates in the json itself does not influence how this function works.

Reply:
If we switch a scraper from CSV to this then shouldn't we just make a new scraper in the array with the date of the switch?

Reply:
Yup

Reply:
Oh I understand what you mean; anything previously using the REST API will have cache misses if we switch them all to this new call...

Reply:
All scrapers were migrated.